sys/shell/commands/shell_commands.c

```diff
@@ -120,6 +120,10 @@ extern int _zep_init(int argc, char **argv);
 extern int _gnrc_rpl(int argc, char **argv);
 #endif
 
+#ifdef MODULE_AT30TSE75X
+extern int _at30tse75x_handler(int argc, char **argv);
+#endif
+
```

Comment:
Would you mind moving that some line upwards, just behind the lines for `MODULE_LTC4150`